email-javamail/src/test/groovy/io/micronaut/email/javamail/JavaMailBodyAndAttachmentSpec.groovy

```diff
@@ -51,8 +52,8 @@ class JavaMailBodyAndAttachmentSpec extends Specification {
         MailhogClient client = applicationContext.getBean(MailhogClient)
 
         and:
-        String from = "sender@here.com"
-        String to = "receiver@here.com"
+        Contact from = new Contact("sender@here.com", "Zapp Brannigan")
+        Contact to = new Contact("receiver@here.com", "Kif")
```

Comment:
do we have a test which verifies it works without passing a name?

Reply:
No, I'll add one now

Reply:
Pushed a tests here https://github.com/micronaut-projects/micronaut-email/pull/59/commits/691f1026de7247a5447488bf0ede38c32a1f7827 🤞

Reply:
thanks